modules/@ergonode/product-templates/src/store/productTemplate/actions.js

```diff
@@ -101,13 +101,10 @@ export default {
                 value: imageID,
             });
 
-            const elementDescriptions = attributes.reduce((prev, curr) => {
-                const tmp = prev;
-
-                tmp[curr.id] = curr.label || curr.code;
-
-                return tmp;
-            }, {});
+            const elementDescriptions = attributes.reduce((prev, curr) => ({
```

Comment:
`reduce((prev, { id, label, code }) => ...` and then `({ ...prev, [id]: label || code })` looks more clear to me

Reply:
Kinda right, but we do have declared some of the variables above in the same scope. We would still need to name it somehow to prevent duplications (eslint rule) and to be constant we should keep the same convention everywhere. Like `{ id: attributeId, code: attributeCode, label: attributeLabel }`